recordatorio_estudio.py

```diff
@@ -17,7 +17,14 @@ def center_window(window, width_percentage=0.25, height_percentage=0.25):
 def on_yes():
     def check_confirmation(event=None):
         if entry.get() == settings["message"][1]:
-            subprocess.run(rf'"{settings["exe_path"]}" --launch-product={settings["launch_product"]} --launch-patchline=live', shell=True)
+            cmd = rf'"{settings["exe_path"]}"'
+
+            if settings["riot_game_args"]["launch_product"] != "":
```

Comment:
Hi Kiana! Thanks again! 
I think this is getting too specific. Let's change to "extra_paths" in this way users can use, for example `exe_path` to launch notepad and if they want to launch steam or riot games they can call the launcher in the exe_path and whatever argument they need in the `extra_path`. Doing it generic this way we will allow users to use this tool even for launchers and exectuable programs we don't even need to think of. Besides it's simpler for us to develop it.

Reply:
Hey! 

No problem... I totally agree with you here, it would be better to create a more simplistic and generic approach! 

I'll get to work on the changes and let you know once committed 😊